### PR TITLE
Add ability to download files from the last 14 days

### PR DESCRIPTION
**Pull request by romidane**

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -7,11 +7,12 @@ module.exports = function Index({ storageService }) {
   router.get('/', async (req, res) => {
     try {
       const file = await storageService.todaysFile();
+      const nomisReports = await storageService.listFiles();
```
Comment (glenjamin) [inline server/routes/index.js]: I think we should put this behind a second page.

File: server/services/storage.js [hunk 2]
```diff
@@ -61,37 +66,70 @@ async function createBlobServiceClient(blobServiceClient) {
   return {
     downloadFile: downloadFile(service),
     todaysFile: todaysFile(service),
+    listFiles: listFiles(service),
   };
 }
 
-// eslint-disable-next-line no-unused-vars
 function listFiles(service) {
+  const date = new Date();
```
Comment (glenjamin) [inline server/services/storage.js]: Need to get dates per-request, not only on startup.

File: server/routes/index.js (hunk 1 above)
Comment (romidane) [reply server/routes/index.js]: Sure can do 👍

File: server/services/storage.js (hunk 2 above)
Comment (romidane) [reply server/services/storage.js]: Good point 🤦‍♂️

File: server/routes/index.js (hunk 1 above)
Comment (romidane) [reply server/routes/index.js]: But it did look cool thanks to the html 5 `<details>` tag 😅

**Pull request merged**